test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -22,6 +27,39 @@ using namespace solidity;
 using namespace dev::solidity::test;
 using namespace std;
 
+namespace
+{
+	/// Translates Solidity's ABI types into the internal type representation of
```

Comment:
We usually do not indent inside namespaces.

Reply:
Thanks.